content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md

```diff
@@ -91,33 +96,70 @@ renv::restore()
 </p>
 
-| File | Usage |
-| ---  |  ---  |
-|.`Rprofile` | File that activates renv for the new R sessions |
-| `renv.lock` | The created lockfile containing all packages and versions |
-| `renv/activate.R` | Activation script run by `.Rprofile` |
-| `renv/library` | Private project library |
-| `renv/settings.json` | Project settings |
+To ensure `renv` is part of your project environment, simply select the *"Use renv with this project"* checkbox. This way, RStudio will automatically configure the necessary settings for `renv`. 
+
+Given this automatic setup, **you can skip step 1** from the workflow mentioned above.
+
+{{% /tip %}}
+
+## Alternative R Dependency Management Options
+
+The R community offers multiple tools to manage dependencies and reproducibility. While `renv` stands out due to its features, other tools have historically been used or still offer specific features that might be appealing to some users.
+
+### Packrat
+
+[Packrat](https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/packrat/) is an earlier solution for managing project-specific R libraries. It helps isolate projects and make them more reproducible by ensuring that each project's library is separate. 
 
-## From packrat to renv
+Nonetheless, while `packrat` was widely used, it has some limitations which `renv` has aimed to address. Furthermore, while it is still maintained, there will be no new development for it. This presents `renv` as a better alternative.
 
-{{% warning %}}
+{{% tip %}}
+**From Packrat to Renv**
 
-[Packrat](https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/packrat/) is now deprecated. To switch from Packrat to renv, use `renv::migrate()` to migrate projects.
+ To switch from `packrat` to `renv`, use `renv::migrate()` to migrate projects.
 
-{{% /warning %}}
+{{% /tip %}}
 
-## Why renv is better than groundhog
+### Groundhog
 
-`groundhog` is another R package that makes projects reproducible. However, it doesn't work by keeping records of packages versions necessary for each project, like `renv` does. Instead you have to manually load libraries with the option of giving a date on when the wanted version was available. Thus, `renv` has more advantages because it can load all necessary packages in one go, without having to manually input the dates certain versions were available on.
+`groundhog` is another R package that facilitates project reproducibility. Unlike `renv` which maintains records of package versions specific to each project, `groundhog` requires users to manually load libraries by specifying a date to fetch the version available at that time.
 
+This can be challenging, as users have to recall the specific dates when certain versions were available. On the other hand, `renv` conveniently loads all necessary packages in one go, making the overall process smoother.
 
+## Additional Resources
```

Comment:
Good to add a "summary" section with key bullets after the additional resources section

Reply:
✅Done

In fact, I've accompanied it with the workflow picture, and it looks quite well.